spec/spec_helper.rb

```diff
@@ -9,6 +9,7 @@
 # in spec/support/ and its subdirectories.
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 
+ActiveRecord::Migration.maintain_test_schema!
```

Comment:
@carlesjove I'm not 100% sure, but I think we can remove `ActiveRecord::Migration.check_pending!` if we use `ActiveRecord::Migration.maintain_test_schema!`, could you please check it? Thanks.